lib/grover/js/processor.cjs

```diff
@@ -17,6 +17,28 @@ const path = require('path');
 const _processPage = (async (convertAction, urlOrHtml, options) => {
   let browser, errors = [], tmpDir;
 
+  const handleErrors = () => {
```

Comment:
not so sure about this method name.. `handleErrors` is a little nondescript. Maybe `handleRequestErrors` ? 

the `errors` variable probably should have been named in a similar fashion!